pkg/webui/account/containers/tokens-table/tokens-table.js

```diff
@@ -38,6 +41,7 @@ import {
 } from '@account/store/selectors/authorizations'
 
 const m = defineMessages({
+  tableTitle: 'Access Tokens',
```

Comment:
```suggestion
  accessTokens: 'Access tokens',
```
[No title case](url).

Reply:
Please also check other messages, I see more title case being used in:
`OAuth Clients` -> `OAuth clients`
`OAuth Client Authorizations` -> `OAuth client authorizations`